### ISSUE TITLE
It would be great if there is a class name on delete entity on the entity box

### ISSUE THREAD

**Issue by ayuwono** [marklogic/marklogic-data-hub#688]
It would be great if there is a class name on delete entity on the entity box so automation on e2e testing is easier

Right now, the span element has no class, and the child is an <i> element that has "fa fa-remove" as the class. It seems that protractor can't recognize it

The edit entity has a class name "edit-start"

```
<span _ngcontent-c10="" class="edit-area">
        <!--bindings={
  "ng-reflect-ng-if": "true"
}--><span _ngcontent-c10="">
          <i _ngcontent-c10="" class="mdi mdi-import"></i>
          1
        </span>
        <!--bindings={
  "ng-reflect-ng-if": "true"
}--><span _ngcontent-c10="">
          <i _ngcontent-c10="" class="mdi mdi-looks"></i>
          1
        </span>

        <span _ngcontent-c10="" class="edit-start">
          <i _ngcontent-c10="" class="fa fa-pencil"></i>
        </span>
        <span _ngcontent-c10="">
          <i _ngcontent-c10="" class="fa fa-remove"></i>
        </span>
      </span>
```

For clicking to edit the entity box, we can use something like:
browser.executeScript('window.document.getElementsByClassName("edit-start")[1].click()');

┆Status: New

**Comment by wooldridge** [comment]
@ayuwono, is the following sufficient?

https://github.com/marklogic/marklogic-data-hub/pull/993

**Comment by wooldridge** [comment]
Merged #993 to address this issue.
